.changeset/itchy-mangos-wink.md

```diff
@@ -0,0 +1,5 @@
+---
+'@verdaccio/middleware': patch
```

Comment:
add @verdaccio/core/url

Reply:
Ohh, thank you! I generated the changeset before realizing I also needed to update the types in that package.
But it's `@verdaccio/url`, right? Not `@verdaccio/core/url`. `@verdaccio/url` is the package name according to the package.json in there

Reply:
It's `@verdaccio/url` you are right